src/test/scripts/product_versions.py

```diff
@@ -45,6 +53,13 @@ def get_lts_version(argv):
 	return lts_version
 
 
+def loadJSON(fdata):
+	result = re.search("\[(.*?)\]", fdata.decode("utf-8"))
+	if result is None:
+		return ""
+	return json.loads(f"[{result.group(1)}]")
```

Comment:
If you use "group 0", it'll use the whole match (and you wouldn't need to use a matching group at all), and you don't need to surround it with `[]`.
```suggestion
	return json.loads(result.group(0))
```